rpm/sailfish-office.spec

```diff
@@ -16,7 +16,7 @@ BuildRequires: cmake
 BuildRequires: qt5-qttools-linguist
 Requires: calligra-components >= 2.7.9+git4
 Requires: calligra-filters >= 2.7.9+git4
-Requires: sailfishsilica-qt5 >= 0.21.63
+Requires: sailfishsilica-qt5 >= 0.23.14
```

Comment:
I don't see that this commit/change would need this requirement. Qt5.6 (quick) on the other hand is required. (I'm seeing indirect reasoning).